src/content/packages-and-plugins/swift-package-manager/for-plugin-authors.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Swift Package Manager for plugin authors
+description: How to add Swift Package Manager compatibility to iOS and macOS plugins
+diff2html: true
+---
+
+Flutter is migrating to Swift Package Manager to manage iOS and macOS native
+dependencies.
+This is an experimental feature that may change in the future.
+It is currently only available on the [`master` channel][].
```

Comment:
consider: 'main'

Reply:
Turns out I was wrong, switched to `main`!